packages/api/deploy/dockerfiles/gnomadgraphqlbase.dockerfile

```diff
@@ -2,22 +2,8 @@ FROM node:7.5.0
 
 MAINTAINER MacArthur Lab
 
-# ENV NODE_ENV=production GRAPHQL_PORT=8000 MONGO_URL=mongodb://localhost:27017/exac
-
-COPY package.json /var/www/
```

Comment:
Switch order of `WORKDIR /var/www` and `COPY package.json /var/www/` because `WORKDIR` doesn't change. Ordering layers from less frequently changed to more frequently changed makes the build cache more useful.